Sources/protoc-gen-grpc-swift/Docs.docc/spm-plugin.md

```diff
@@ -0,0 +1,113 @@
+# Using the Swift Package Manager plugin
```

Comment:
A lot of this is copied from the protobuf documentation, can we just link to their docs where possible?

Reply:
Don't we prefer to have "self-contained" docs for each package/plugin? In theory this plugin and the protobuf plugin can evolve separately.
I don't mind making this change, but just curious. I've also ended up making a few changes in this PR that should be copied over (or moved) to the protobuf repo, so I should open a PR to `swift-protobuf`.

Reply:
I don't think we have a strong stance. I'm okay with leaving this as is. There's some value in not punting folks over to a different resource. The cost is diverging docs for common things (installing protoc) but we can bear that.